spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/DefaultSubscriberFactory.java

```diff
@@ -155,16 +159,27 @@ public void setApiClock(ApiClock apiClock) {
 
 	/**
 	 * Set the retry settings for the generated subscriber stubs.
-	 * @param subscriberStubRetrySettings parameters for retrying pull requests when they fail,
-	 * including jitter logic, timeout, and exponential backoff
+	 * @param subscriberStubRetrySettings parameters for retrying pull requests when they
+	 * fail, including jitter logic, timeout, and exponential backoff
 	 */
-	public void setSubscriberStubRetrySettings(RetrySettings subscriberStubRetrySettings) {
+	public void setSubscriberStubRetrySettings(
+			RetrySettings subscriberStubRetrySettings) {
 		this.subscriberStubRetrySettings = subscriberStubRetrySettings;
 	}
 
+	/**
+	 * Set the (n)acknowledgement of messages to be asynchronous
+	 * @param asyncAck
```

Comment:
The parameter must have some description.

Let's turn on a Checkstyle rule on the matter: http://checkstyle.sourceforge.net/config_javadoc.html#NonEmptyAtclauseDescription

Reply:
Created https://github.com/spring-cloud/spring-cloud-gcp/issues/869 since this issue occurs in various places within the project.